src/disk_interface.cc

```diff
@@ -284,3 +284,9 @@ void RealDiskInterface::AllowStatCache(bool allow) {
     cache_.clear();
 #endif
 }
+
+void RealDiskInterface::ClearStatCache() {
+#ifdef _WIN32
```

Comment:
Why?

Reply:
It seems like the cached timestamps would become stale if other processes touched the files after the server was started. I added this to invalidate the cache it at the start of each build.

Reply:
Why is that the case only on Windows?

Reply:
The cache_ member variable only exists on Windows. All other functions dealing with the cache have #ifdef _WIN32 as well, such as the previous function in this file.

Reply:
I see. Can you wrap ClearStatCache in `#ifdef _WIN32` including the call instead of the function body?

Reply:
Done.